pytest.ini

```diff
@@ -14,17 +14,21 @@ filterwarnings=
 	# Ensure ResourceWarnings are emitted
 	default::ResourceWarning
 
+	# Workarounds for pypa/setuptools#3810
+	# Can't use EncodingWarning as it doesn't exist on Python 3.9
+	default:'encoding' argument not specified
+	# subprocess.check_output still warns with EncodingWarning even with encoding set
+	ignore:'encoding' argument not specified::setuptools.tests.environment
+
 	# realpython/pytest-mypy#152
 	ignore:'encoding' argument not specified::pytest_mypy
 
-	# python/cpython#100750
-	ignore:'encoding' argument not specified::platform
-
-	# pypa/build#615
-	ignore:'encoding' argument not specified::build.env
+	# pytest-dev/pytest # TODO: Raise issue upstream 
+	ignore:'encoding' argument not specified::_pytest
 
-	# dateutil/dateutil#1284
-	ignore:datetime.datetime.utcfromtimestamp:DeprecationWarning:dateutil.tz.tz
+	# pypa/distutils # TODO: Raise issue upstream 
+	ignore:'encoding' argument not specified::distutils
+	ignore:'encoding' argument not specified::setuptools._distutils
```

Comment:
These changes shouldn't go in "## upstream" - those are handled in [jaraco/skeleton](/jaraco/skeleton).  Let's add any setuptools-specific ones below "## end upstream".

Reply:
Since `setuptools._distutils` is vendored, I understood "upstream" as "this warning is raised from code that would need to be changed upstream".